### DISCUSSION TITLE
Drop `TwitterLegacy`

### DISCUSSION THREAD

**Discussion by CodedOre** [CodedOre/NewCaw#44] [Backend]
librest has recently [removed their OAuth 1.0 support](https://gitlab.gnome.org/GNOME/librest/-/commit/a36606b6633e4244740793ab3a17d392b1cf6035), so this leaves us without backend for `TwitterLegacy`.

For now I have kept it in development because the 2.0 API for Twitter did not had anything we needed, mainly the home timeline and video support. The first is now there (need to check on the second), so maybe we should drop `TwitterLegacy` from NewCaw?

**Comment by IBBoard** [comment]
Feels like there's not much of a discussion in this one!

"Do we use early libhandy or later libhandy or libadwaita?" was about whether enough of our users were going to be on a new enough platform soon enough. This is about whether we're going to be putting effort into a system that's slowly disappearing.

The only minor issue with dropping legacy Twitter is that the v2 APIs aren't all there yet. But that just means that we have to delay until they are.

**Comment by CodedOre** [comment]
So, `TwitterLegacy` is now removed.

I also checked quickly in the API reference for Twitters 2.0 API, and it seems that the only thing still missing an endpoint is the profile banners, and out-of-band on authentication.
